meta-facebook/aalc-rpu/src/platform/plat_hwmon.c

```diff
@@ -415,7 +418,13 @@ uint8_t pwm_control(uint8_t group, uint8_t duty)
 	static int64_t auto_tune_time;
 	if (!get_status_flag(STATUS_FLAG_AUTO_TUNE)) {
 		auto_tune_time = k_uptime_get();
-		set_pwm_group(group, 0);
+		return 0;
+	}
+
+	// keep 0 duty when first enabling auto tune
+	static bool is_init = false;
+	if (!is_init) {
+		is_init = true;
```

Comment:
does it make more sense to move this down to line 439 once we know that things are initialized?

Reply:
Based on the reply of the next commit, if the duty is set after line 439, the pump will start when the system is ready